src/components/App/App.js

```diff
@@ -17,7 +25,15 @@ class App extends Component {
         <nav>
           <h1>Rancid Tomatillos</h1>
         </nav>
-        <Movies movies={this.state.movies}/>
+        {/* {Object.keys(this.state.selectedMovie).length && */}
```

Comment:
Would a ternary work better in this instance? 
{Object.keys(this.state.selectedMovie).length 
   ?  <Details />
   :  <Movies />